templates/corporate/hello.html

```diff
@@ -1,562 +1,402 @@
-{% extends "zerver/portico.html" %}
-{% set entrypoint = "landing-page" %}
-
-{% set PAGE_TITLE = "Zulip: Open-source team chat with topic-based threading" %}
-
-{% set PAGE_DESCRIPTION = "Zulip is the only modern team chat app that is
-  designed for both live and asynchronous conversations. Organized to help you
-  collaborate productively and efficiently." %}
-
+{% extends "zerver/base.html" %}
+{% set entrypoint = "landing-page-hello" %}
 {% block customhead %}
-<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
-<style>
-    .portico-page {
-    padding-bottom: 0px;
-    }
-</style>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>
+    <title>Zulip — organized team chat</title>
+    <!-- Google / search engine tags -->
+    <meta name="description" content="Zulip is the only modern team chat app that is designed for both live and asynchronous conversations. Organized to help you collaborate productively and efficiently." />
+    <!-- Open Graph / Facebook / Twitter meta tags -->
+    <meta property="og:url" content="https://zulipchat.com/" />
+    <meta property="og:type" content="website" />
+    <meta property="og:site_name" content="Zulip" />
+    <meta property="og:title" content="Zulip: Open-source team chat with topic-based threading" />
+    <meta property="og:description" content="Zulip is the only modern team chat app that is designed for both live and asynchronous conversations. Organized to help you collaborate productively and efficiently." />
+    <meta property="og:image" content="https://zulipchat.com/static/images/logo/zulip-icon-128x128.png" />
```

Comment:
We should use zulip.com rather than zulipchat.com in all the URLs in this PR, I think.

Reply:
Done.